specification/redisenterprise/resource-manager/Microsoft.Cache/preview/2022-11-01-preview/redisenterprise.json

```diff
@@ -0,0 +1,1971 @@
+          },
+          {
+            "$ref": "#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "x-ms-examples": {
+          "SkusList": {
+            "$ref": "./examples/RedisEnterpriseSkusList.json"
+          }
+        },
+        "responses": {
+          "200": {
+            "description": "The request succeeded, response body contains the requested details.",
+            "schema": {
+              "$ref": "#/definitions/RegionSkuDetails"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Sku": {
+      "description": "SKU parameters supplied to the create RedisEnterprise operation.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The type of RedisEnterprise cluster to deploy. Possible values: (Enterprise_E10, EnterpriseFlash_F300 etc.)",
+          "enum": [
+            "Enterprise_E10",
+            "Enterprise_E20",
+            "Enterprise_E50",
+            "Enterprise_E100",
+            "EnterpriseFlash_F300",
+            "EnterpriseFlash_F700",
+            "EnterpriseFlash_F1500"
+          ],
+          "x-ms-enum": {
+            "name": "SkuName",
+            "modelAsString": true
+          }
+        },
+        "capacity": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The size of the RedisEnterprise cluster. Defaults to 2 or 3 depending on SKU. Valid values are (2, 4, 6, ...) for Enterprise SKUs and (3, 9, 15, ...) for Flash SKUs."
+        }
+      },
+      "required": [
+        "name"
+      ]
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Current provisioning status",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "Creating",
+        "Updating",
+        "Deleting"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningState",
+        "modelAsString": true
+      }
+    },
+    "ResourceState": {
+      "type": "string",
+      "readOnly": true,
+      "description": "Current resource status",
+      "enum": [
+        "Running",
+        "Creating",
+        "CreateFailed",
+        "Updating",
+        "UpdateFailed",
+        "Deleting",
+        "DeleteFailed",
+        "Enabling",
+        "EnableFailed",
+        "Disabling",
+        "DisableFailed",
+        "Disabled",
+        "UnavailableLostCmkAccess"
```

Comment:
>UnavailableLostCmkAccess [](http://example.com/codeflow?start=9&length=24)

@jantache-microsoft Is this actually going to be something we return any time soon? Today, I wonder a) if we need it. b) whether we might want something more flexible that can return other unavailability reasons.

Reply:
Good point, let's remove it for now, lacking any concrete plans to do extra monitoring around this, and also lacking a concrete plan for using a generic unavailability state value.